packages/rpc-provider/package.json

```diff
@@ -26,17 +26,17 @@
   },
   "homepage": "https://github.com/polkadot-js/api/tree/master/packages/rpc-provider#readme",
   "dependencies": {
-    "@babel/runtime": "^7.4.5",
+    "@babel/runtime": "^7.5.0",
     "@polkadot/storage": "^0.82.0-beta.29",
-    "@polkadot/util": "^0.94.0-beta.4",
-    "@polkadot/util-crypto": "^0.94.0-beta.4",
+    "@polkadot/util": "^0.94.0-beta.5",
+    "@polkadot/util-crypto": "^0.94.0-beta.5",
     "@types/nock": "^10.0.3",
     "eventemitter3": "^4.0.0",
     "isomorphic-fetch": "^2.2.1",
-    "websocket": "^1.0.28"
+    "websocket": "^1.0.29"
```

Comment:
Not sure why these guys thing that `gulp` is a "dependency" (instead of dev dep).